varya/assets/sass/base/_editor.scss

```diff
@@ -50,6 +50,16 @@ a {
 		font-family: var(--global--font-secondary);
 		font-size: var(--global--font-size-md);
 	}
+
```

Comment:
The `.alignleft` and `.alignright` utility classes are all located in `/sass/blocks/utilities/_style.scss`. If this isn’t fixed in #85, can you move this improvement there?

Reply:
I checked out #85 and the issue was still there.

I'll move these styles over to `/sass/blocks/utilities/_style.scss` 👍